system-addon/test/unit/content-src/components/ContextMenu.test.js

```diff
@@ -0,0 +1,57 @@
+const React = require("react");
+const {shallow} = require("enzyme");
+const ContextMenu = require("content-src/components/ContextMenu/ContextMenu");
+const DEFAULT_PROPS = {
+  onUpdate: () => {},
+  visible: false,
+  options: [],
+  tabbableOptionsLength: 0
+};
+
+describe("<ContextMenu>", () => {
+  it("shoud be hidden by default", () => {
+    const wrapper = shallow(<ContextMenu {...DEFAULT_PROPS} />);
+    assert.isTrue(wrapper.find(".context-menu").props().hidden);
+  });
+  it("should be visible if props.visible is true", () => {
+    const props = Object.assign({}, DEFAULT_PROPS, {visible: true});
```

Comment:
Up to you whether you want to change it for this file but for the future, I find this is a little more succinct:
```js
const wrapper = shallow(<ContextMenu {...DEFAULT_PROPS} visible={true} />);
```